scripts/report_scripts/index.Rmd

```diff
@@ -489,178 +531,250 @@ if (nrow(mutations_sig_unfiltered) > 0) {
 }
 ```
 
-```{r text_mutations_pooled, results='asis', eval=APPROVED_MUTATIONS_FOUND}
+```{r filter_pool_mutations, eval=RUN_MUTATION_REGRESSION}
+# filter good samples for only mutations with sig pvalues for plotting
+filtered_approved_mut_plot <- dplyr::select(
+  approved_mut_plot,
+  c(
+    samplename,
+    dates,
+    location_name,
+    coordinates_lat,
+    coordinates_long,
+    mutations_sig$mutation
+  )
+)
 
-cat('## Trending mutations over time {.tabset .tabset-fade}
+# check if the filtered df has actual values besides meta data
+APPROVED_MUTATIONS_FOUND <- (length(filtered_approved_mut_plot) > 5) &&
+  (nrow(filtered_approved_mut_plot) > 0)
 
-These plots show the relative frequency of detected mutations with strong increasing trends in samples at specific wastewater sampling locations and how the frequencies change over time. Please use the tabs to access the not-pooled plots for each location.')
+approved_mut_plot_location_pooled <- pool_by_mean(filtered_approved_mut_plot,
+  na_handling = TRUE,
+  group_fun = "day_location"
+)
 ```
 
+```{r mutation_trends_pooled_text, results='asis', eval=APPROVED_MUTATIONS_FOUND}
+cat(
```

Comment:
Maybe this is in a code block, because of the `eval` up there...?  Conditional display?  In any case, seems kinda clunky with all that R string noise.

Reply:
Yes, all that display code in code chunks is largely there for the conditional execution, except for the one in `variantreport_p_sample.Rmd`, there it just didn't occur to me to take it out of the code chunk. What would be a good alternative to still display the text only under certain conditions?

Reply:
I think it would be nicer to use the `asis` engine, not the `r` engine.  E.g. https://stackoverflow.com/a/66454300

Reply:
great that you found this! the `asis` engine sounds like a very handy tool, and we should probably adapt it for the other pipelines as well.

Reply:
I am going to do that for this pipeline. But should the warnings and notes also be converted to `asis`? Currently that works via functions, but those are not very complex (yet), but I wouldnt know how to work with functions in pure html...